server/app/services/DateConverter.java

```diff
@@ -32,14 +32,24 @@ public String formatIso8601Date(Instant instant) {
     return DateTimeFormatter.ISO_DATE.withZone(zoneId).format(instant);
   }
 
+  /**
+   * Converts a DateString in "YYYY-MM-dd" format to LocalDate type Throws DateTimeParseException
+   */
+  public LocalDate parseStringToLocalDate(String dateString) {
```

Comment:
suggestion (blocking): please match the naming convention of the other method
```suggestion
  public LocalDate parseIso8601DateToLocalDate(String dateString) {
```

Reply:
Done! thank you. I also updated the documentation.